docs/self-managed/connectors-deployment/connectors-configuration.md

```diff
@@ -223,75 +223,71 @@ java -cp 'connector-runtime-application-VERSION-with-dependencies.jar:...:my-sec
 
 ## Multi-tenancy
 
-The Connector Runtime supports multiple tenants for inbound and outbound Connectors.
+The Connector Runtime supports multiple tenants for inbound and outbound Connectors. These are configurable in [Identity](/self-managed/identity/user-guide/tenants/managing-tenants.md).
+
 A single Connector Runtime can serve a single tenant or can be configured to serve
-multiple tenants. By default, the runtime uses the `<default>` tenant ID for all
-Zeebe related operations like handling Jobs and publishing Messages.
+multiple tenants. By default, the runtime uses the default tenant ID for all
+Zeebe-related operations like handling jobs and publishing messages.
 
 :::info
 Support for **outbound Connectors** with multiple tenants requires a dedicated
-tenant job worker config (described below). **Inbound Connectors** will automatically work for all tenants
-the configured Connector Runtime client has access to. This can be configured in Identity via
-the application assignment.
+tenant job worker config (described below). **Inbound Connectors** automatically work for all tenants the configured Connector Runtime client has access to. This can be configured in Identity via the application assignment.
 :::
 
 ### Environment variables
 
-The following environment variables are used by the Connector Runtime
-for the configuration of multi-tenancy.
+The Connector Runtime uses the following environment variables to configure multi-tenancy:
 
-| Name                                       | Description                                                     | Default value |
-| ------------------------------------------ | --------------------------------------------------------------- | ------------- |
-| ZEEBE_CLIENT_DEFAULT-TENANT-ID             | The default tenant ID used to communicate with Zeebe            | `<default>`   |
-| ZEEBE_CLIENT_DEFAULT-JOB-WORKER-TENANT-IDS | The default tenants IDs (comma separated) used to activate jobs | `<default>`   |
+| Name                                       | Description                                                    | Default value |
+| ------------------------------------------ | -------------------------------------------------------------- | ------------- |
+| ZEEBE_CLIENT_DEFAULT-TENANT-ID             | The default tenant ID used to communicate with Zeebe           | `myTenant`    |
+| ZEEBE_CLIENT_DEFAULT-JOB-WORKER-TENANT-IDS | The default tenant IDs (comma separated) used to activate jobs | `myTenant`    |
```

Comment:
Same as above, we can be explicit about the default being `<default>` here, but this might benefit from updating the description if my understanding is correct? 

```suggestion
| ZEEBE_CLIENT_DEFAULT-TENANT-ID             | The default tenant ID used to communicate with Zeebe. Changing this value will set a new default tenant ID used for fetching jobs and publishing messages .          | `myTenant`    |
| ZEEBE_CLIENT_DEFAULT-JOB-WORKER-TENANT-IDS | The default tenant IDs (comma separated) used to activate jobs. To run the Connector Runtime in a setup where a single runtime serves multiple tenants, add each tenant ID to this list. | `myTenant`    |
```